tests/test_converted_equivalence.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2022 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
```

Comment:
@nikalra What is the different between test_converted_equivalence and test_equivalence? Should/can these be in the same file?

Reply:
test_equivalence compares the results of our stft/melspec layers against librosa/torchaudio. test_converted_equivalence compares the results of the Torch version of our implementation against the converted implementation.

I opted to keep them separate because test_equivalence is platform agnostic, whereas test_converted_equivalence must be run on the target hardware for the given on-device framework. More than happy to combine the two files if you feel like that'd be better/cleaner.

Reply:
Sounds good, no need to change.